src/exception/ErrorResponse.ts

```diff
@@ -0,0 +1,44 @@
+import { Request } from 'express';
+import { BusinessException } from './BusinessException';
+
+/**
+ * Class to represent custom HTTP error messages.
+ */
+export class ErrorResponse {
```

Comment:
Se for uma exception, então sufixar a classe com Exception, como nas demais. Caso essa classe não seja uma exception, então remover da pasta onde esta se encontra.